src/models/advisor.ts

```diff
@@ -14,8 +14,8 @@ export class AdvisorInsight implements AdvisorInsightInterface {
 
   constructor(insight: AdvisorInsightInterface) {
     switch (insight.level as InsightLevel) {
```

Comment:
Since the `InsightLevel` values match the i18n string ids, this switch could be eliminated.

Reply:
Updated.